server/plugin.go

```diff
@@ -208,11 +208,13 @@ func (p *Plugin) start(syncSince *time.Time) {
 
 	if p.getConfiguration().SyncUsers > 0 {
 		p.API.LogDebug("Starting the sync users job")
-		p.syncUsers()
 
 		// Close the previous background job if exists.
 		p.stopSyncUsersJob()
 
+		// Start a syncing of users on plugin start. The below job just schedules the job to run at a given interval of time but does not run it while scheduling. To avoid this, we call the function once separately to sync the users.
```

Comment:
```suggestion
		// Start syncing the users on plugin start. The below job just schedules the job to run at a given interval of time but does not run it while scheduling. To avoid this, we call the function once separately to sync the users.
```